addons/mass_mailing/models/mailing.py

```diff
@@ -291,6 +292,16 @@ def _compute_next_departure(self):
             else:
                 mass_mailing.next_departure = cron_time
 
+    @api.depends('next_departure', 'state')
+    def _compute_next_departure_is_past(self):
+        ''' If the departure time has passed but mailing is still in queue, this compute field
```

Comment:
we use double quote for docstring :) 

```suggestion
        """ If the departure time has passed but mailing is still in queue, this compute field
```